drafts/a11y-crosswalk-MARC/index.html

```diff
@@ -4,6 +4,7 @@
 <head>
 	<meta charset="utf-8" />
 	<title>Accessibility Properties Crosswalk (schema.org, ONIX, MARC21 &amp; UNIMARC)</title>
+    <meta http-equiv="refresh" content="0; URL=https://w3c.github.io/a11y-discov-vocab/crosswalk/"/>
```

Comment:
If you want to give people some time to read the page, the 0 here is the number of seconds for the browser to wait before redirecting. You could change it to 10, for example, and add a comment to the body that you will be automatically redirected in 10s.